### DISCUSSION TITLE
What are the calculate at compile time solutions measuring?

### DISCUSSION THREAD

**Discussion by fvbakel** [PlummersSoftwareLLC/Primes#536] [Q&A] [answered]
I was looking into the solutions that seem to do the sieve calculation at compile time, like `PrimeCPP/solution_3`. I was wondering what is actually calculated in each pass that is measured? Is there even done any calculation at all during the measurement or is it just some kind of memory copy? What are we measuring in these solutions?

**Comment by rbergen** [comment, marked as answer]
I think what's measured will depend on the exact language and the implementation; there's actually a rather detailed discussion to be found on this in #502. We clearly don't include these solutions as contestants in the leaderboard (which is documented by them being marked as unfaithful), but as demonstrations of the oddities that are indeed intrinsic to some of the languages in the collection.

**Reply by fvbakel** [reply]
@rbergen Ah, I will look in to that. I actually never heard of these techniques until now so it is indeed a nice addition to this project.

**Reply by rbergen** [reply]
If you feel my response answered your question, then please feel free to mark it as such :) It will show the question as answered in the Q&A topic list which may help others who have the same question.
